webpack/webpack.min.config.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 const webpack = require('webpack');
 const banner = require('./banner');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
since we are using webpack 4 I think this is included by default in the [mode](https://webpack.js.org/concepts/mode/) flag

Reply:
Yes, you are right. However, by using this plugin we include more options. With this plugin we go from 244k to 241k, which is not a lot so I think we can discuss whether to use it or not.